os/arch/arm/src/s5j/s5j_spi.c

```diff
@@ -446,7 +453,14 @@ static void spi_exchange(struct spi_dev_s *dev, const void *txbuffer, void *rxbu
 		while (received < nwords) {
 			if (sent < nwords)
 				if (SPI_STAT_TX_FIFO_LVL(getreg32(&pSPIRegs->SPI_STATUS)) < 64) {
-					putreg32(((unsigned char *)txbuffer)[sent++], &pSPIRegs->SPI_TX_DATA);
+					if (fifo_width == 1) 
```

Comment:
This update is for SPI driver.
In order to correctly handle bus width setting configuration value is checked. 
Not it can handle 8/16/32 bit transfers.

Reply:
To improve readability, why don't you use define?
For example,
#define SPI_FIFO_WIDTH_8_BIT  (8 >> 3)
#define SPI_FIFO_WIDTH_16_BIT  (16 >> 3)
#define SPI_FIFO_WIDTH_32_BIT  (32 >> 3)

Reply:
Could you please add bracket for single line?? it is our coding style..